### PR TITLE
Refactor Volume group

### PR DESCRIPTION
**Pull request by ismirlia**
```
--- PASS: TestAccIBMPIVolumeGroupUpdate (499.98s)
PASS
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_volume_group.go [hunk 1]
```diff
@@ -35,51 +35,78 @@ func ResourceIBMPIVolumeGroup() *schema.Resource {
...
 			},
-			"replication_status": {
-				Type:        schema.TypeString,
+			Attr_ReplicationStatus: {
 				Computed:    true,
 				Description: "Volume Group Replication Status",
+				Type:        schema.TypeString,
 			},
-			"consistency_group_name": {
+			Attr_StatusDescriptionErrors: {
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_volume_group.go:76]: Can you please add `status_description_errors` in docs?

Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group.go:76]: added description

File: ibm/service/power/resource_ibm_pi_volume_group.go [hunk 2]
```diff
@@ -131,20 +158,20 @@ func resourceIBMPIVolumeGroupRead(ctx context.Context, d *schema.ResourceData, m
...
-	d.Set("volume_group_status", vg.Status)
-	d.Set("consistency_group_name", vg.ConsistencyGroupName)
-	d.Set("replication_status", vg.ReplicationStatus)
-	d.Set(PIVolumeGroupName, vg.Name)
+	d.Set(Arg_VolumeGroupName, vg.Name)
 	d.Set(Arg_VolumeIDs, vg.VolumeIDs)
-	d.Set("status_description_errors", flattenVolumeGroupStatusDescription(vg.StatusDescription.Errors))
+	d.Set(Attr_ConsistencyGroupName, vg.ConsistencyGroupName)
+	d.Set(Attr_ReplicationStatus, vg.ReplicationStatus)
+	d.Set(Attr_StatusDescriptionErrors, flattenVolumeGroupStatusDescription(vg.StatusDescription.Errors))
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_volume_group.go:172]: Can you please add a null check for `vg.StatusDescription`?

File: website/docs/r/pi_volume_group.html.markdown [hunk 3]
```diff
@@ -43,29 +46,37 @@ ibm_pi_volume_group provides the following [timeouts](https://www.terraform.io/d
...
-- `id` - (String) The unique identifier of the volume group. The ID is composed of `<pi_cloud_instance_id>/<volume_group_id>`.
 - `consistency_group_name` - (String) The consistency Group Name if volume is a part of volume group.
+- `id` - (String) The unique identifier of the volume group. The ID is composed of `<pi_cloud_instance_id>/<volume_group_id>`.
 - `replication_status` - (String) The replication status of volume group.
+- `status_description_errors` - (Set) The status details of the volume group.
+  
+  Nested scheme for `status_description_errors`:
+  - `key` - (String) The volume group error key.
+  - `message` - (String) The failure message providing more details about the error key.
+  - `volume_id` - (List of String) List of volume IDs, which failed to be added to or removed from the volume group, with the given error.
```
Comment (yussufsh) [inline website/docs/r/pi_volume_group.html.markdown:70]:
```suggestion
  - `volume_ids` - (List of String) List of volume IDs, which failed to be added to or removed from the volume group, with the given error.
```

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: ibm/service/power/resource_ibm_pi_volume_group.go (hunk 2 above)
Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group.go]: made changes

File: website/docs/r/pi_volume_group.html.markdown (hunk 3 above)
Comment (ismirlia) [reply website/docs/r/pi_volume_group.html.markdown]: made changes

**Review by yussufsh — APPROVED** [review]
lgtm

**Pull request merged**